dev-docs/source/ReleaseChecklist.rst

```diff
@@ -138,7 +138,18 @@ language, layout, and collecting images.
 Wednesday, 2 weeks & 3 days
 ---------------------------
 
-*  Create issues for people to neaten up the release notes and add images etc.
+*  Switch to manual handling of release notes by running the release_editor.py script using the
+   `release editor helper tool
+   <https://github.com/mantidproject/mantid/blob/main/tools/ReleaseNotes/release_editor.py>`_
+   and open a pull request to put them on ``release``. The script copies all of the separate release notes under the correct heading of their upper level file, e.g. framework.rst, and moves
```

Comment:
Just a minor correction here as I assume ``release`` should be the name of the release branch: ``release-next``?